R/ppmify.R

```diff
@@ -302,6 +305,36 @@ grid <- function (area, density) {
 
 }
 
+bias <- function (area, bias_points, offset.raster=FALSE){
+  
+  # calculate weights for integration points when using pseudo-observations as integration points
+  # weight = total study area / number of integration/quadrature points
+  
+  if (offset.raster){ # if offset raster is also provided, calculate sum of raster values
+    
+    a <- cellStats(area, sum, na.rm=TRUE)  
```

Comment:
I get the bias points version, but not the implementation of bias raster.

For bias raster, I was imagining using the grid method for creating the integration points, but instead of giving each integration point the total area of pixels in its grid cell, it would get the total amount of effort in the pixels in it's grid cell (the sum of the values in the bias raster). This is just getting one value over the entire raster, right?

At the moment, the bias raster is only providing a total amount of effort, not the spatial distribution in effort. So that will change the value of the predictions by some multiplicative factor, but not account for bias.